src/bundle/Resources/public/scss/_alerts.scss

```diff
@@ -80,4 +83,98 @@
         background-color: $ibexa-color-success-100;
         border-color: $ibexa-color-success-300 $ibexa-color-success-300 $ibexa-color-success-300 $ibexa-color-success;
     }
+
+    &--toast {
+        padding: calculateRem(12px);
+
+        &.ibexa-alert {
+            &--info {
+                border-radius: calculateRem(10px);
+                border: calculateRem(1px) solid $ibexa-color-info;
+
+                .ibexa-alert {
+                    &__icon-wrapper {
+                        border-radius: 50%;
+                        background: $ibexa-color-info;
+                    }
+                }
+            }
+
+            &--error {
+                border-radius: calculateRem(10px);
+                border: calculateRem(1px) solid $ibexa-color-danger;
+
+                .ibexa-alert {
+                    &__icon-wrapper {
+                        border-radius: $ibexa-border-radius-small;
+                        background: $ibexa-color-danger;
+                        transform: rotate(45deg);
+
+                        .ibexa-icon {
+                            transform: rotate(-45deg);
+                        }
+                    }
+                }
+            }
+
+            &--warning {
+                border-radius: calculateRem(10px);
+                border: calculateRem(1px) solid $ibexa-color-accent;
+
+                .ibexa-alert {
+                    &__icon-wrapper {
+                        position: relative;
+
+                        &:before {
+                            display: inline-block;
+                            margin-left: calculateRem(3px);
+                            font-size: calculateRem(5px);
+                            border: solid calculateRem(15px) transparent;
+                            border-top-width: 0;
+                            border-bottom-width: calculateRem(25px);
+                            border-bottom-color: $ibexa-color-accent;
+                            clip-path: circle(54% at 50% 69%);
+                            vertical-align: bottom;
+                            content: '';
+                        }
+
+                        .ibexa-icon {
+                            position: absolute;
+                            top: calculateRem(9px);
+                            left: calculateRem(8px);
+                        }
+                    }
+                }
+            }
+
+            &--success {
+                border-radius: calculateRem(10px);
```

Comment:
Can we add these three (success warning and error) to some common selector, as they have some same properties